portfolio.css

```diff
@@ -0,0 +1,245 @@
+*{
+	box-sizing: border-box;
+	color:#fff;
+}
+
+/* Background image css */
+
+body{
+	background-image: url(mountain_bg.jpg);
+	background-repeat: no-repeat;
+	background-position: bottom;
+	background-color:#FBEEC1;
+	background-attachment:fixed;
+	background-size:cover;
+}
+
+/* Header formatting */
+
+h1, h2{
+	background-color:#115464;
+	border-radius:16px;
+	border-style: solid;
+	border-width:2px;
+	border-color:#0c96b0;
+	margin:auto;
+	margin-top:10px;
+	margin-bottom:10px;
+	padding:5px;
+	font-size: 30px;
+}
+
+.projects h1{
+	margin-bottom:5px;
+}
+
+h3{
+	display:inline;
+	background-color:#f7ebae;
+	border-radius:16px;
+	border-style: solid;
+	border-width: 2px;
+	border-color:#228db1;
+	margin: 10px auto;
+	padding:5px;
+	font-size: 30px;
+	color:#000;
+}
+
+/* Hyperlink formatting */
+
+a, a:active{
+	font-size:25px;
+	padding:5px;
+	border-radius:16px;
+	border-style: solid;
+	border-width:2px;
+	border-color:#f1db91;
+	margin:0 auto;
+	color:#fff;
+	text-decoration:none;
+	background-color:#228db1;
+	outline:none;
+}
+
+a:hover {
+	background-color:#f1db91
+}
+
+a:focus{
+	background-color:#f1db91
+}
+
+a:visited{
+	background-color:#228db1;
+}
+
+nav a{
+	font-size:calc(20px + 0.3vw);
+	margin:8px auto;
+}
+
+.contact_buttons a{
+	font-size:4vw;
+	margin: 10px auto
+}
+
+/* moves the top link to the left of the page */
+
+.top_link{
+	margin:0px ;
+}
+
+/*formatting on the left, right and center text sections and initial home section*/
+
+section{
+	width:80%;
+	margin:0 auto;
+	display:flex;
+	justify-content:space-between;
+}
+
+.paragraph{
+	flex-direction: column;
+	justify-content: start;
+	background-color:#12505f;
+	border-radius:16px;
+	border-style: solid;
+	border-width:2px;
+	border-color:#f7ebae;
+	margin:50px auto;
+	padding:20px;
+}
+.paragraph p{
+	width:60%;
+}
+
+.paragraph p, .center_bar p{
+	font-size:calc(20px + 0.5vw);
+}
+
+.center_bar{
+	display:flex;
+	margin:0 auto;
+	width:80%;
+	flex-direction:column;
```

Comment:
Don't think this is doing anything?

Reply:
this overrides the justify content on the section tag :)